src/components/molecules/NavBar/NavBar.tsx

```diff
@@ -240,8 +265,7 @@ export const NavBar: React.FC<NavBarPropsType> = ({ hasBackButton = true }) => {
               ) : (
                 <div>{navbarTitle}</div>
               )}
-
-              <VenuePartygoers venueId={venueId} />
+              <PlayaTime /> - <VenuePartygoers venueId={venueId} />
```

Comment:
```suggestion
              <PlayaTime />{" "}-{" "}<VenuePartygoers venueId={venueId} />
```

Or a combo of these ( https://www.w3.org/MarkUp/HTMLPlus/htmlplus_13.html ):
```
&ndash; // en dash (half the width of an em unit)
&mdash; // em dash (equal to width of an "m" character)
&ensp; // en space
&emsp; // em space
&nbsp; // non breaking space
```

Reply:
fixed in 0700a1d, had to use a div with styles as empty brackets were removed by react upon interpretation